### PR TITLE
init RemoteDatasource

### PR DESCRIPTION
**Pull request by kpt96**
init RemoteDatasource 
Link redmine: https://edu-redmine.framgia.vn/issues/1206

### CODE REVIEW

**Comment by kpt96** [comment]
ready

**Comment by kpt96** [comment]
ready

File: app/src/main/java/com/framgia/moviedb_23/screen/home/HomeFragment.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        HomeFragment homeFragment = new HomeFragment();
+        return homeFragment;
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
+                             Bundle savedInstanceState) {
+//        View v = inflater.inflate(R.layout.fragment_home, container, false);
+//        return v;
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/moviedb_23/screen/home/HomeFragment.java]: 25 26 remove comment

**Comment by kpt96** [comment]
ready

**Pull request merged**